new-content/learning-resources/textbook.md

```diff
@@ -4,9 +4,9 @@ to: /textbook
 image: /images/landing-page/feature-hardware.jpg
```

Comment:
This comment is for the previous line: `to: /textbook`

If we use as URL `/textbook` instead of `https://qiskit.org/texbook`, the `AppLink` component classifies it as "no external" link and converts it into a `nuxt-link`. 
<img width="780" alt="Screen Shot 2020-07-16 at 10 29 34" src="https://user-images.githubusercontent.com/17231966/87646522-414b0200-c74f-11ea-9bd4-22c21128e0ff.png">

When we click on it, we get an error.
<img width="545" alt="Screen Shot 2020-07-16 at 10 30 09" src="https://user-images.githubusercontent.com/17231966/87646650-56c02c00-c74f-11ea-8748-00229b867e14.png">

Reply:
Second error is normal, the textbook is not mounted during development time.

Regarding the one, I'm fine adding the complete URL to force it to be open is a new tab.

Reply:
I'll fix this when reviewing #749